### DISCUSSION TITLE
58. VATZ biweekly meeting at `2024-06-26 10:30:00 +09:00:00 (KST)`

### DISCUSSION THREAD

**Discussion by xellos00** [dsrvlabs/vatz#573] [General]
### 1. Overall
<img width="1247" alt="image" src="https://github.com/dsrvlabs/vatz/assets/6308023/baaa63fa-86a8-4fd3-826f-16f7926679db">

### 2. Statistic Rate
 
Sprint | Issue fulfillment | progress rate(%)
--: | :--: | :--:

<img width="1251" alt="image" src="https://github.com/dsrvlabs/vatz/assets/6308023/43c1239f-130c-4ac7-912e-dd27ba8a7672">

---

# Announcements
- https://github.com/dsrvlabs/validators/issues/1161

**Comment by xellos00** [comment]
# DK
- https://github.com/dsrvlabs/vatz/issues/561
- https://github.com/dsrvlabs/vatz/issues/556
   - https://github.com/dsrvlabs/vatz/pull/569 
- https://github.com/dsrvlabs/vatz/issues/567
- https://github.com/dsrvlabs/vatz/issues/568
- https://github.com/dsrvlabs/vatz/issues/572
- https://github.com/dsrvlabs/vatz-plugin-watcher-cosmos/issues/5

**Comment by gnongs** [comment]
## Jinhong
- https://github.com/dsrvlabs/vatz/issues/560
